tests/drivers/dma/chan_blen_transfer/boards/nucleo_f746zg.conf

```diff
@@ -1,3 +1,2 @@
 CONFIG_DMA_TRANSFER_CHANNEL_NR_0=0
 CONFIG_DMA_TRANSFER_CHANNEL_NR_1=1
-CONFIG_NOCACHE_MEMORY=y
```

Comment:
On H7, it was required to do:
```
 # Required by BDMA which only has access to do:
# a NOCACHE SRAM4 section. All other DMAs also
# has access to this section.
CONFIG_CODE_DATA_RELOCATION=y
CONFIG_DMA_LOOP_TRANSFER_RELOCATE_SECTION="SRAM4"
```

Reply:
due to the BDMA using the SRAM4, this section has to be declared as NOCACHE
For testing with stm32F7, the overlay is setting all the sram0 as NOCACHE